src/node/transaction.cpp

```diff
@@ -14,12 +14,12 @@
 
 #include <future>
 
-TransactionError BroadcastTransaction(const CTransactionRef tx, std::string& err_string, const CAmount& max_tx_fee, bool relay, bool wait_callback)
+TransactionError BroadcastTransaction(const CTransactionRef tx, std::string& err_string, const CAmount& max_tx_fee, bool relay, bool wait_callback, CConnman* connman)
```

Comment:
I guess I'd prefer to pass in the node here, so that in the future when mempool and chainstate are members of the node struct, this doesn't need to change again.

```suggestion
TransactionError BroadcastTransaction(const CTransactionRef tx, std::string& err_string, const CAmount& max_tx_fee, bool relay, bool wait_callback, Node& node)
```